### PR TITLE
Addition of Horse Barding

### PR DESCRIPTION
**Pull request by Arthrutus**
Added Script for Horse barding. Majority of files compatible with latest release. Issue when restarting server and chopping an already barded horse. Horse will disappear. Issue seems to stem from horse.cs but may stem from the other scripts. Fix needed.

### CODE REVIEW

**Comment by kamronbatman** [comment]
Looks good. I'll help with the bug tomorrow.
1. Let's delete the modifications file
1. Change the readme to README.md

**Comment by Arthrutus** [comment]
Done!

File: Scripts/HorseBarding/MODIFICATIONS [hunk 1]
```diff
@@ -0,0 +1 @@
+Items/Internal/InvisibleTile.cs git@github.com:ServUO/ServUO.git/01f18a3eff59847e1dd3856137c969c37f77649e/Scripts/Items/Internal/InvisibleTile.cs
```
Comment (kamronbatman) [inline Scripts/HorseBarding/MODIFICATIONS]: Delete this file.

File: Scripts/HorseBarding/Migrations/Server.Items.InvisibleTile.v0.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (kamronbatman) [inline Scripts/HorseBarding/Migrations/Server.Items.InvisibleTile.v0.json]: Delete this file.

**Pull request merged**